web/packages/extension/assets/_locales/en/messages.json

```diff
@@ -20,6 +20,9 @@
     "settings_log_level": {
         "message": "Log level"
     },
+    "settings_max_execution_duration": {
+        "message": "How long can actionscript execute for before being forcefully stopped (in seconds)"
```

Comment:
I missed that :)
actionscript -> ActionScript

Reply:
I completely forgot about it as well :sweat_smile: .